src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -8256,31 +8256,37 @@ export class StudyViewPageStore
         default: {},
     });
 
-    readonly getDataForClinicalDataTab = remoteData({
-        await: () => [this.clinicalAttributes, this.selectedSamples],
+    readonly getDataForClinicalDataTab: MobxPromiseUnionTypeWithDefault<
```

Comment:
seems odd that you have to type this?  like, i would think we would be able to use api types as before?

Reply:
I have removed this type definition.